README.md

```diff
@@ -28,5 +28,7 @@ ACT-R has conference calls every 4 weeks. If you are interested in becoming an a
 
```

Comment:
Not specifically related to the purpose of this PR, but you may wish to update the conference call frequency sentence at the same time (not really worth a separate PR).

Reply:
I think that's better done in a separate PR.